src/box/sql/vdbe.h

```diff
@@ -299,14 +299,8 @@ void sqlVdbeComment(Vdbe *, const char *, ...);
 #define VdbeComment(X)  sqlVdbeComment X
 void sqlVdbeNoopComment(Vdbe *, const char *, ...);
 #define VdbeNoopComment(X)  sqlVdbeNoopComment X
-#ifdef SQL_ENABLE_MODULE_COMMENTS
```

Comment:
There is one mention of `SQL_ENABLE_MODULE_COMMENTS` left in vdbe.h, could be dropped too.

Reply:
Thanks, dropped.